### PR TITLE
Create order admin panel in custom user admin.

### PR DESCRIPTION
**Pull request by kjcioffi**
* Provided store owners with the ability to:
   * View their store orders.
   * Update their store orders.
   * Delete their store orders.

### CODE REVIEW

File: store/forms.py [hunk 1]
```diff
@@ -14,4 +13,9 @@ class ProductAdminForm(forms.ModelForm):
...
         model = Product
         exclude = ["store"]
-    
\ No newline at end of file
+
+
+class OrderAdminForm(forms.ModelForm):
+    class Meta:
+        model = Order
+        exclude = ["store", "products"]
```
Comment (ryaustin) [inline store/forms.py:21]: should "total_cost" also be excluded?

**Review by ryaustin — APPROVED** [review]
Looks good Kevin:
 - consider if total_cost should be an editable field
 - _maybe for another version_, allow modification of product quantities, removing products or adding products to an existing order.

File: store/forms.py (hunk 1 above)
Comment (kjcioffi) [reply store/forms.py:23]: I like this. I didn't exclude it from the form but I did edit the widget to make the form control read-only.

**Comment by kjcioffi** [comment]
> Looks good Kevin:
> 
> * consider if total_cost should be an editable field
> * _maybe for another version_, allow modification of product quantities, removing products or adding products to an existing order.

Thanks, Ryan!

**Pull request merged**